projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -56,7 +57,17 @@ export default function PublicizeForm( {
 		useSocialMediaConnections();
 	const { message, updateMessage, maxLength } = useSocialMediaMessage();
 	const { isEnabled: isSocialImageGeneratorEnabledForPost } = useImageGeneratorConfig();
+	const { dismissedNotices, dismissNotice } = useDismissNotice();
 
+	const hasInstagramConnection =
+		connections.filter( connection => connection.service_name === 'instagram' ).length > 0;
```

Comment:
We could use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here.

```
connections.some( connection => connection.service_name === 'instagram' )
```